gammapy/stats/variability.py

```diff
@@ -137,3 +138,84 @@ def compute_chisq(flux):
     yobs = flux.data
     chi2, pval = stats.chisquare(yobs, yexp)
     return chi2, pval
+
+
+def compute_flux_doubling(flux, flux_err, coords, axis=0):
+    r"""Compute the minimum characteristic flux doubling and halving
+    over a certain coordinate axis for a series of measurements.
+
+    Computing the flux doubling can give the doubling time in a lightcurve
+    displaying significant temporal variability, e.g. an AGN flare.
+
+    The variable is computed as:
+
+     .. math::
+        doubling = min(\frac{t_(i+1)-t_i}{log_2{f_(i+1)/f_i}})
+
+    Where f_i and f_(i+1) are the fluxes measured at subsequent coordinates t_i and t_(i+1)
+    The error is obtained by propagating the relative errors on the flux measures.
+
+    Parameters
+    ----------
+    flux : `~astropy.units.Quantity`
+        the measured fluxes
+    flux_err : `~astropy.units.Quantity`
+        the error on measured fluxes
+    coords: `~astropy.units.Quantity`
+        the coordinates at which the fluxes are measured
+    axis : int, optional
+        Axis along which the value is computed.
+
+    Returns
+    -------
+    doubling, doubling_err, coord : `~numpy.ndarray`
```

Comment:
In fact, the function returns 3 ndarrays, not one.
The first ndarray contains doubling and halving flux, the second their corresponding errors, and the 2 coordinate values at which it happens